docs/src/main/tut/index.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: home
+title:  "ML Lambda"
+permalink: 'index.html'
+position: 1
+---
+
+## What is ML Lambda? 
+
+__ML Lambda__ is an opensource machine learning serving cluster for deploying and monitoring machine learning models and ad-hoc algorithms. This enables you get your models up and running in an instant, on just about any infrastructure and using any of the available machine learning toolkits. It also gives you an opportunity to monitor your model's performance, analyse inputs of the model, like, determine whether there'is an increased number of outliers recently or not, observe model's decisions on given data and so on. 
```

Comment:
__ML Lambda__ is an opensource model management cluster for deploying, serving and monitoring machine learning models and ad-hoc algorithms.